scripting/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,66 @@
+= Scripting Module - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: connectors::scripting/scripting-module.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Scripting Module v2.0
+
+The Scripting Module executes custom logic written in a scripting language. In some cases, you might need to create custom code to perform all or part of a complex task, or to reuse modules that you have already written.
+Since version 2.0 you must provide a compliant `jsr-223` scripting language engine.
+
+For information about compatibility and fixed issues, see xref:release-notes::mule-runtime/module-scripting.adoc[Scripting Module Release Notes]
+
+== Prerequisites
+
+To use this module, you must be familiar with the following concepts:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow
+* Global elements
+* Understand how to create a Mule app using Design Center or Anypoint Studio
+
+Before creating an app, you must:
```

Comment:
Since there is just one prereq, you can combine and just make one sentence here. 
Before creating an app, you must have access to Anypoint Platform.